container/docker/handler.go

```diff
@@ -82,6 +83,24 @@ type dockerContainerHandler struct {
 	fsHandler fsHandler
 }
 
+func getRwLayerID(containerID, storageDriverDir string, dockerVersion []int) (string, error) {
+	const (
+		// Docker version >=1.10.0 have a randomized ID for the root fs of a container.
+		randomizedRWLayerMinorVersion = 10
+		// Directory where the file containinig the randomized ID of the root fs of a container is stored in versions >= 1.10.0
+		rwLayerIDDir  = "../image/aufs/layerdb/mounts/"
+		rwLayerIDFile = "mount-id"
+	)
+	if !(dockerVersion[0] >= 1 && dockerVersion[1] >= randomizedRWLayerMinorVersion) {
```

Comment:
Should this be || instead of &&?
for docker 2.0 -> [2, 0] -> !(true && false) -> !(false) -> true